src/llama_stack_client/lib/cli/common/utils.py

```diff
@@ -3,15 +3,28 @@
 #
 # This source code is licensed under the terms described in the LICENSE file in
 # the root directory of this source tree.
-from tabulate import tabulate
```

Comment:
does this mean `requirements.txt` should change?

Reply:
also, what is the `TqdmExperimentalWarning` about?

Reply:
Ah yes, updated requirements.txt

> also, what is the TqdmExperimentalWarning about?

seems like warning from using rich's tqdm, disabled it now